src/datasources/queues/queues-api.module.ts

```diff
@@ -25,8 +25,17 @@ function queueConsumerFactory(
     configurationService.getOrThrow<string>('amqp.exchange.mode');
   const queue = configurationService.getOrThrow<string>('amqp.queue');
   const prefetch = configurationService.getOrThrow<number>('amqp.prefetch');
+  const heartbeatIntervalInSeconds = configurationService.getOrThrow<number>(
+    'amqp.heartbeatIntervalInSeconds',
+  );
+  const reconnectTimeInSeconds = configurationService.getOrThrow<number>(
+    'amqp.reconnectTimeInSeconds',
+  );
```

Comment:
Although I would prefer explicitly setting the default in our config, should we default to `null`, this generic is incorrect:

```suggestion
  const heartbeatIntervalInSeconds = configurationService.getOrThrow<number | null>(
    'amqp.heartbeatIntervalInSeconds',
  );
  const reconnectTimeInSeconds = configurationService.getOrThrow<number | null>(
    'amqp.reconnectTimeInSeconds',
  );
```

Reply:
Already Fixed in a recent change. https://github.com/safe-global/safe-client-gateway/pull/2572/commits/247f9773c2be34488e62bc9cbd58092e38cd279a